packages/core/contracts/chainbridge/SourceGovernor.sol

```diff
@@ -24,9 +35,12 @@ contract SourceGovernor is Ownable {
 
     /**
      * @notice This is the first method that should be called in order to relay a governance request to another network
-     * marked by `sinkChainID`. Note: this can only be called by the owner (presumably the L1 governor).
+     * marked by `destinationChainId`. Note: this can only be called by the owner (presumably the L1 governor).
```

Comment:
- The `relayGovernance` [function comment](https://github.com/mrice32/protocol/blob/5e81b20b13a961cab8f54b89d2e145a946cd9619/packages/core/contracts/chainbridge/SourceGovernor.sol#L32) says `sinkChainID` instead of `destinationChainId`.